Index: main.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/main.py b/venv/main.py
rename from main.py
rename to venv/main.py
--- a/main.py	(revision dbfc96be7ac33f1307254696379016406d28058f)
+++ b/venv/main.py	(date 1681762509392)
@@ -1,4 +1,4 @@
-from utils import load_data, filter_sort, formatted_data
+from functions import load_data, filter_sort, formatted_data
 
 
 def main():
diff --git a/operations.json b/venv/operations.json
rename from operations.json
rename to venv/operations.json
diff --git a/utils.py b/venv/functions.py
rename from utils.py
rename to venv/functions.py
